backend/src/data/repositories/interview/interview.repository.ts

```diff
@@ -38,6 +40,44 @@ class Interview {
     return interview ?? null;
   }
 
+  public create({
+    status,
+    categoryId,
+    intervieweeUserId,
+  }: InterviewsCreateRequestDto): Promise<InterviewM> {
+    return this.#InterviewModel
+      .query()
+      .insert({
+        status,
+        categoryId,
+        intervieweeUserId,
+      })
+      .execute();
+  }
+
+  public getPendingOrPassedInterviewsByUserId(
+    intervieweeUserId: number,
+  ): Promise<InterviewM[]> {
+    return this.#InterviewModel
+      .query()
+      .where({ intervieweeUserId })
+      .andWhere('status', '!=', InterviewStatus.REJECTED)
```

Comment:
Won't it be suitable to use?
```suggestion
      .andWhereNot('status', InterviewStatus.REJECTED)
```